edb/server/protocol/pg_ext.pyx

```diff
@@ -70,27 +70,34 @@ def managed_error():
 @cython.final
 cdef class ConnectionView:
     def __init__(self):
-        self._settings = DEFAULT_SETTINGS
-        self._fe_settings = DEFAULT_FE_SETTINGS
         self._in_tx_explicit = False
         self._in_tx_implicit = False
+
+        # Kepp track of backend settings so that we can sync to use different
+        # backend connections (pgcon) within the same frontend connection,
+        # see serialize_state() below and its usages in pgcon.pyx.
+        self._settings = DEFAULT_SETTINGS
         self._in_tx_settings = None
+
+        # Frontend-only settings are defined by the high-level compiler, and
```

Comment:
High-level compiler meaning the `edb/compiler/compiler.py`, `compile_sql` function?

Reply:
ya the one you just moved to `edb/server/compiler/sql.py` ;)